code/components/nui-core/src/NUIInitialize.cpp

```diff
@@ -429,7 +429,13 @@ void HookLibGL(HMODULE libGL)
 {
 	MH_Initialize();
 	MH_CreateHook(GetProcAddress(libGL, "glTexParameterf"), glTexParameterfHook, (void**)&g_origglTexParameterf);
-	MH_CreateHook(GetProcAddress(LoadLibraryW(L"d3d11.dll"), "D3D11CreateDevice"), D3D11CreateDeviceHook, (void**)&g_origD3D11CreateDevice);
+
+	auto d3d11Handle = LoadLibraryW(L"d3d11.dll");
+	if (!d3d11Handle)
+	{
+		FatalError("CitizenFX: Cannot load 'd3d11.dll'. Please (re)install 'Direct X' for your OS.");

```

Comment:
Not sure if this needs an error state since CitiLaunch takes in `d3d11.dll` already as well as ViabilityTests->DXGICheck in CitiLaunch doing this as well.

The error phrasing is also a tad odd - `Direct X` -> `DirectX`, and it is typically part of the OS itself nowadays (unlike d3dx*.dll).